ibl_to_nwb/metadata/ecephys.yml

```diff
@@ -82,3 +82,7 @@ Ecephys:
       description: Proportion of quality metrics passed.
     - name: allen_location
       description: Brain region reference in the Allen Mouse Brain Atlas.
+    - name: cluster_uuid
+      description: Unique identifier given to each cluster after spike sorting. Enables comparison of each cluster in this .nwb file to its counterpart in ONE.
+    - name: cluster_id
+      description: The original id given to each cluster after spike sorting. Note that as spike sorting is computed per probe, cluster_ids are only unique per probe insertion, experiments with dual insertions can have clusters with the same id.
```

Comment:
I guess one question here though, why have both `cluster_uuid` and `cluster_id`? Isn't the UUID sufficient?

Reply:
I agree that in terms of information, the uuid is sufficient ;)  - I put the `cluster_id` back in there because it was actually in there before but I had removed as I replaced the index with [consecutive numbering](https://github.com/int-brain-lab/IBL-to-nwb/blob/4f936247959737f7fcb1b7dd49dfe7423be600a4/ibl_to_nwb/datainterfaces/iblsortingextractor.py#L51). Somehow it reflects also something that is a bit more human readable than the uuid. I guess this only really matters if people split the units table by probe insertion, and then the index also remains as a human readable format.

Reply:
Hmm I see!

Will make final decision once we see some new example files then